modules/monitoring/manifests/checks.pp

```diff
@@ -109,6 +109,26 @@
   }
   # END signon
 
+  # START search
+  # TODO: increase the comparison period to a week so alerts don't dissapear if they happen over the weekend
+  # and extend checks to other indices
+  icinga::check::graphite { 'check_rummager_govuk_index_size_increased':
+    target    => 'divideSeries(stats.gauges.govuk.app.rummager.govuk_index.docs.count, timeShift(stats.gauges.govuk.app.rummager.govuk_index.docs.count, "1d"))',
+    warning   => 110,
+    critical  => 125,
+    desc      => 'rummager govuk index size has increased by more than 10% over the last 3 days',
+    host_name => $::fqdn,
+  }
+
+  icinga::check::graphite { 'check_rummager_govuk_index_size_decreased':
+    target    => 'transformNull(divideSeries(timeShift(stats.gauges.govuk.app.rummager.govuk_index.docs.count, "1d"), stats.gauges.govuk.app.rummager.govuk_index.docs.count), 10000000)',
```

Comment:
What's the significance of 10000000? And why doesn't the other check need `transformNull`?

Reply:
Sorry, I should read the whole of the PR description...

Reply:
There is a definite case I want this check to handle where the count goes from some number to zero. In this case I put 10000000 in as an arbitrary value.

The other case is checking for an increase, so the only divide by zero is if the data was wrong in the past. In this case we can't tell if something's wrong so I'm just ignoring it.

I'm not really happy with this though because there's also the case where graphite has null data - this will currently also trigger the "decreased" alert.